acceptance/framework/vault/helpers.go

```diff
@@ -6,63 +6,11 @@ import (
 	"fmt"
 	"testing"
 
```

Comment:
This diff is not the easiest to follow unfortunately.  The best way to review this is to first gain an understanding of the vault helper functions are used in the tests.  So look to the test files first and look for `vault.*` function calls.